File: app/src/main/java/ch/threema/app/preference/SettingsChatFragment.java
/*  _____ _
 * |_   _| |_  _ _ ___ ___ _ __  __ _
 *   | | | ' \| '_/ -_) -_) '  \/ _` |_
 *   |_| |_||_|_| \___\___|_|_|_\__,_(_)
 *
 * Threema for Android
 * Copyright (c) 2018-2021 Threema GmbH
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU Affero General Public License, version 3,
 * as published by the Free Software Foundation.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 * GNU Affero General Public License for more details.
 *
 * You should have received a copy of the GNU Affero General Public License
 * along with this program. If not, see <https://www.gnu.org/licenses/>.
 */

package ch.threema.app.preference;

import android.os.Bundle;
import androidx.annotation.Nullable;
import android.view.View;

import ch.threema.app.R;

public class SettingsChatFragment extends ThreemaPreferenceFragment {

	@Override
	public void onCreatePreferencesFix(@Nullable Bundle savedInstanceState, String rootKey) {
		addPreferencesFromResource(R.xml.preference_chat);
	}

	@Override
	public void onViewCreated(View view, Bundle savedInstanceState) {
		preferenceFragmentCallbackInterface.setToolbarTitle(R.string.prefs_chatdisplay);
		super.onViewCreated(view, savedInstanceState);
	}

}

File: app/src/main/java/ch/threema/app/preference/SettingsChatFragment.kt
package ch.threema.app.preference

import ch.threema.app.R
import ch.threema.app.utils.logScreenVisibility
import ch.threema.base.utils.getThreemaLogger

private val logger = getThreemaLogger("SettingsChatFragment")

@Suppress("unused")
class SettingsChatFragment : ThreemaPreferenceFragment() {
    init {
        logScreenVisibility(logger)
    }

    override fun getPreferenceTitleResource(): Int = R.string.prefs_chatdisplay

    override fun getPreferenceResource() = R.xml.preference_chat
}
